### PR TITLE
Create include-product-owners.md

### PR DESCRIPTION
**Pull request by NewMexicoKid**
This was created at the Spring Summit 2017 in Geneva, Switzerland.

### CODE REVIEW

**Comment by spier** [comment]
@NewMexicoKid about this pattern: 
Do you know if this pattern is already proven (state)?
Also would you mind adding a Patlet to it?

Maybe the most important question:
Are you able to work on this pattern, or should I reach out to somebody else, or as the last resort should we close this PR and keep the pattern only as a draft?

**Comment by spier** [comment]
Hi @NewMexicoKid (mentioning you as you spent quite some time on this PR already). 

We have reworked the [maturity levels][levels] of the InnerSource patterns. The new levels make it more explicit what quality standards are expected for each level. With that we will also try to get PRs merged into master faster.

**Status of this pattern:**

We have currently categorized the Pattern in this PR as maturity level **Level 2 - Structured**.

For that level we have two requirements:

* is validated by at least one known instance
   * we suspect that this is already true, as this pattern was previously listed in the group `Pattern Drafts (proven, not yet fully reviewed)`
* complies with the [patterns format][format]
   * not sure about this, especially as the pattern format may have evolved since this PR was opened

**How to get this PR merged?**

* Could you confirm if the status assessment above sounds correct to you?
* If this pattern does not match the [patterns format][format] yet, could you adapt it accordingly?
* With that we should be able to merge this PR to master. That will also allow other contributors to level up this pattern further in subsequent PRs/contributions in the future.

**I have no time to work on this, what to do?**

As this PR is open for a while, it would be completely understandable if your priorities have shifted and you don't time to work on this anymore.

In that case please just let us know in a comment below. Then we can decide what to do with this PR on our own.

Thank you for your help! 👍 

[levels]: https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/meta/contributor-handbook.md#maturity-levels
[format]: https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/meta/pattern-template.md

**Comment by spier** [comment]
@NewMexicoKid I made a number of changes to align this pattern with the Pattern Template and to move it into our new maturity levels.

I decided to move it to Level 1 (Initial) for now.

Things missing to move it to Level 2:

* Patlet
* Expanded explanation of the Known Instance. Based on the current description I am not sure if that Known Instance was actually implemented or rather an idea? (See "PayPal is looking into finding a search solution" [link](https://github.com/InnerSourceCommons/InnerSourcePatterns/blame/0fdf59db6346748ea58a4f602041b48c6ad0ab2e/patterns/1-initial/include-product-owners.md#L60))
* Cross-referencing and linking it to other patterns that may have been created in the meantime.

Given that this pattern has been sitting here as a PR for a long time, my suggestion would be to merge this pattern as Level 1 (Initial). Then we can wait and see if gets confirmed and improved by others in the community.

What do you think?

File: patterns/1-initial/include-product-owners.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+## Title
+
+Include Product Owners
+
+## Patlet
+
+TBD
```
Comment (NewMexicoKid) [inline patterns/1-initial/include-product-owners.md]: Engaging and educating Product Owners about InnerSource can help them modify their actions (e.g., in the space of KPIs) to help InnerSource collaboration work better.

**Review by NewMexicoKid — COMMENTED** [review]
Changes look okay (I added a suggestion for a patlet).

File: patterns/1-initial/include-product-owners.md (hunk 1 above)
Comment (spier) [inline patterns/1-initial/include-product-owners.md]:
```suggestion
Engaging and educating Product Owners about InnerSource can help them modify their actions (e.g., in the space of KPIs) to help InnerSource collaboration work better.
```

Comment (spier) [reply patterns/1-initial/include-product-owners.md]: Awesome, adding that in.

File: README.md [hunk 2]
```diff
@@ -77,6 +76,7 @@ The below lists all known patterns. They are grouped into three [maturity levels
 * [Code Consumers](patterns/1-initial/code-consumers.md)
 * [Explaining InnerSource to Management by anchoring it to Agile / DevOps / Lean](patterns/1-initial/concept-anchor.md)
 * [Reluctance to Receive Contributions](patterns/1-initial/reluctance-to-accept-contributions.md) - *Core owner of shared asset is reluctant to take contributions due to the required maintenance that comes with them. Summary pattern that lays out four children patterns with three to be defined.*
+* [Include Product Owners](patterns/1-initial/include-product-owners.md) - *Key Performance Indicators (KPIs) for Product Owners are primarily product focused, and don't consider areas such as collaborative development. This results in a lower level of engagement with InnerSource projects.*
```
Comment (spier) [inline README.md]:
```suggestion
* [Include Product Owners](patterns/1-initial/include-product-owners.md) - *Engaging and educating Product Owners about InnerSource can help them modify their actions (e.g., in the space of KPIs) to help InnerSource collaboration work better.*
```

**Pull request merged**